----------------------------------------------------------------
Mon Jun 01 21:49:26 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance a816c00e-0172-7281-25cc-000011725d00 
on database directory C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@18b4aac2 
Loaded from file:/C:/Users/Gustavo/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Mon Jun 01 21:50:56 BRT 2020:
Shutting down instance a816c00e-0172-7281-25cc-000011725d00 on database directory C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@18b4aac2 
Mon Jun 01 21:50:57 BRT 2020 Thread[Thread-202,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Mon Jun 01 21:50:57 BRT 2020 Thread[Thread-202,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Mon Jun 01 21:50:57 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 00cd0029-0172-7281-25cc-000011725d00 
on database directory C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@18b4aac2 
Loaded from file:/C:/Users/Gustavo/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Mon Jun 01 21:53:17 BRT 2020:
Shutting down instance 00cd0029-0172-7281-25cc-000011725d00 on database directory C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@18b4aac2 
Mon Jun 01 21:53:18 BRT 2020 Thread[Thread-399,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Mon Jun 01 21:53:18 BRT 2020 Thread[Thread-399,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Mon Jun 01 21:53:19 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 0b3e4052-0172-7281-25cc-000011725d00 
on database directory C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@18b4aac2 
Loaded from file:/C:/Users/Gustavo/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Mon Jun 01 21:53:53 BRT 2020:
Shutting down instance 0b3e4052-0172-7281-25cc-000011725d00 on database directory C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@18b4aac2 
Mon Jun 01 21:53:53 BRT 2020 Thread[Thread-596,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Mon Jun 01 21:53:53 BRT 2020 Thread[Thread-596,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Mon Jun 01 21:53:54 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 65f6406f-0172-7281-25cc-000011725d00 
on database directory C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@18b4aac2 
Loaded from file:/C:/Users/Gustavo/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Mon Jun 01 21:54:34 BRT 2020:
Shutting down instance 65f6406f-0172-7281-25cc-000011725d00 on database directory C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@18b4aac2 
Mon Jun 01 21:54:34 BRT 2020 Thread[Thread-793,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Mon Jun 01 21:54:34 BRT 2020 Thread[Thread-793,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Mon Jun 01 21:54:35 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 513b408a-0172-7281-25cc-000011725d00 
on database directory C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@18b4aac2 
Loaded from file:/C:/Users/Gustavo/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Mon Jun 01 21:55:59 BRT 2020:
Shutting down instance 513b408a-0172-7281-25cc-000011725d00 on database directory C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@18b4aac2 
Mon Jun 01 21:55:59 BRT 2020 Thread[Thread-990,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Mon Jun 01 21:55:59 BRT 2020 Thread[Thread-990,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Mon Jun 01 21:56:00 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance a0d440e2-0172-7281-25cc-000011725d00 
on database directory C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@18b4aac2 
Loaded from file:/C:/Users/Gustavo/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Mon Jun 01 21:56:47 BRT 2020:
Shutting down instance a0d440e2-0172-7281-25cc-000011725d00 on database directory C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@18b4aac2 
Mon Jun 01 21:56:48 BRT 2020 Thread[Thread-1187,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Mon Jun 01 21:56:48 BRT 2020 Thread[Thread-1187,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Mon Jun 01 21:56:49 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance c0a0c103-0172-7281-25cc-000011725d00 
on database directory C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@18b4aac2 
Loaded from file:/C:/Users/Gustavo/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
----------------------------------------------------------------
Mon Jun 01 21:56:57 BRT 2020:
Shutting down instance c0a0c103-0172-7281-25cc-000011725d00 on database directory C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@18b4aac2 
Mon Jun 01 21:56:58 BRT 2020 Thread[Thread-1384,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
Mon Jun 01 21:56:58 BRT 2020 Thread[Thread-1384,5,main] Cleanup action starting
ERROR XBM0I: Directory 181010_180375 cannot be removed.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection$6.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.impl.jdbc.EmbedConnection.removePersistentService(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$1.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.apache.derby.jdbc.InternalDriver.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver$LoginCallable.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Cleanup action completed
----------------------------------------------------------------
Mon Jun 01 21:56:58 BRT 2020:
Booting Derby version The Apache Software Foundation - Apache Derby - 10.14.2.0 - (1828579): instance 4fccc11e-0172-7281-25cc-000011725d00 
on database directory C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch\181010_180375 with class loader sun.misc.Launcher$AppClassLoader@18b4aac2 
Loaded from file:/C:/Users/Gustavo/.m2/repository/org/apache/derby/derby/10.14.2.0/derby-10.14.2.0.jar
java.vendor=Oracle Corporation
java.runtime.version=1.8.0_241-b07
user.dir=C:\Users\Gustavo\Pictures\POO2\Final-01\gustavo_nogueira-jean_bauch
os.name=Windows 10
os.arch=amd64
os.version=10.0
derby.system.home=null
Database Class Loader started - derby.database.classpath=''
